styles.css

```diff
@@ -240,3 +240,7 @@ tr {
     margin-top: -29px;
   }
 }
+
+a {
+  word-break: break-all;
+}
```

Comment:
i fully understand you, I will fix styling tags directly in few days. but for now is the best choice, at least proper amount of links will be affected